standard/expressions.md

```diff
@@ -798,10 +791,23 @@ An *unfixed* type variable `Xᵢ` *depends directly on* an *unfixed* type varia
 
 `Xₑ` *depends on* `Xᵢ` if `Xₑ` *depends directly on* `Xᵢ` or if `Xᵢ` *depends directly on* `Xᵥ` and `Xᵥ` *depends on* `Xₑ`. Thus “*depends on*” is the transitive but not reflexive closure of “*depends directly on*”.
 
+#### §input-type-inference Input type inferences
+
+An *input type inference* is made *from* an expression `E` *to* a type `T` in the following way:
+
+- If `E` is a tuple expression (§12.8.6) with arity `N` and elements `Eᵢ`, and `T` is a tuple type with arity `N` with a corresponding element type `Tₑ` or `T` is a nullable value type `T0?` and `T0` is a tuple type with arity `N` that has a corresponding element type `Tₑ`, then for each `Eᵢ`, an input type inference is made from `Eᵢ` to `Tₑ`.
```

Comment:
I'm getting lost here - partly because of the single/multiple aspects. Should this:

> `T` is a tuple type with arity `N` with a corresponding element type `Tₑ`

be

> `T` is a tuple type with arity `N` with a corresponding element types `Tₑ`

(Note the plural.) That then matches the first part. about `Eᵢ`. I don't know whether we expand this more elsewhere, with effectively "where 1 <= i <= N" or similar.

Reply:
Yes, it should be plural.